lib/Controller/MfazonesController.php

```diff
@@ -140,7 +158,7 @@ public function set($source, $protect)
 
             $type = $this->castObjectType($node->getType());
 
-            if ($protect == "true") {
+            if ($protect === "true") {
```

Comment:
I think this fixes https://git.radicallyopensecurity.com/ros/pen-sunet-mfa-zones/-/issues/3